spec/models/vmdb_database_connection_spec.rb

```diff
@@ -1,16 +1,15 @@
 require "concurrent/atomic/event"
 
-describe VmdbDatabaseConnection do
-  before do
-    @db = FactoryGirl.create(:vmdb_database)
```

Comment:
yay, this isn't needed and is actually a problem now that disabled transactions for these examples.